.github/workflows/bench.yml

```diff
@@ -36,5 +36,9 @@ jobs:
         continue-on-error: true
         with:
           key: docker-images-${{ runner.os }}-${{ steps.hash-docker-images.outputs.result }}
+      - name: Build
+        run: yarn build
+      - name: Relink
+        run: yarn install
```

Comment:
Why was this necessary?

Reply:
Bench workflows started failing with import errors to hive gateway :/